src/test/java/com/github/fishio/TestPlayerFish.java

```diff
@@ -92,9 +94,43 @@ public void testCollideWithLargerEnemyFish() {
 				null, 0.0, 0.0));
 		when(ef.getBoundingArea().getSize()).thenReturn(6.1);
 		
+		//Set the amount of lives to 1
+		pf.livesProperty().set(1);
+		
+		//Collide with an enemy
+		pf.onCollide(ef);
+		
+		//The player should have no more lives left.
+		assertEquals(0, pf.getLives());
+		
+		//The player should be dead.
+		assertTrue(pf.isDead());
+		
+		Mockito.verify(ef, never()).setDead();
```

Comment:
I only added this to be in line with the testcode already in place (your code). I do, however, agree that it might be better to use a different testing method.

For now, I will add a TODO tag, and add that to a next sprint.

<layout,testing